concepts/default-enarx-trust-process.md

```diff
@@ -0,0 +1,474 @@
+# TBD: Default Enarx trust process
+- Authors: [Mike Bursell](mike@p2ptrust.org)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-03-17
+- Status Note: under discussion
+- Start Date: 2020-03-17 (date you started working on this idea)
+- Tags: trust
+- Addresses: (Issue #)[https://github.com/enarx/enarx/issues/]
+
+TODO: this is curently listed as a "concepts" document, but also
+includes design information and implementation requirements.  Need
+to decide where it best sits.  Neither "concepts" nor "features"
+(as described in the README.md) fit very well.
+
+## Summary
+
+Setting up trust domains and being able to transfer certain entities
+between them in central to Enarx.  This RFC describes the default
+trust process for Enarx and lays out sets of requirements for
+implementations.
+
+## Motivation
+
+One of the key aims of the Enarx project is to allow clients to run
+workloads on hosts which are "untrusted".  This RFC builds on previous
+RFCs to describe the default model by which Enarx allows this take
+place, using the capabilities of TEEs, the underlying CPU(s) and
+firmware of the host and other parties.
+
+It should be noted that the key party for whom trust is considered
+important within the Enarx project is the workload owner - in this case,
+the tenant.  It is trust relationships from this entity to others, and
```

Comment:
I've tried to clarify this.  Please let me know if I've succeeded!

Reply:
It's certainly *more* clear now, but I'm still not sure. If what matters is trust *from me to you*, does that mean it's important that I trust you or that you trust me? I can extrapolate this from the context since I already know what's supposed to be going on, but it takes me a minute to think about this phrasing. Could be just me, though.

Reply:
I've taken some suggested text from @axelsimon which I think helps.